### ISSUE TITLE
Additional patterns for github/gitblit

### ISSUE THREAD

**Issue by markiewb** [markiewb/nb-git-open-in-external-repoviewer#24]
* git://github.com/madflow/flow-netbeans-markdown.git
* http://git.delwink.com/git/r/patts-qt.git

**Comment by 2mac** [comment]
As the author of patts-qt, I'm curious how my repository location ended up here.

I'm not upset, but this page is the third DuckDuckGo result when searching "patts-qt", and I'm just wondering why it's mentioned here. This issue is pretty vague.

**Comment by markiewb** [comment]
I only searched for a public gitserver to test the patterns without setting up my own server. So I used an URL pointing to your server, which i found by using a search engine. If you like I will alter the text body of the issue description and remove your URL. 

Best regards, markiewb

----- Ursprungligt meddelande -----
Från: "David McMackins II" <notifications@github.com>
Skickat: ‎14.‎11.‎2015 15:43
Till: "markiewb/nb-git-open-in-external-repoviewer" <nb-git-open-in-external-repoviewer@noreply.github.com>
Kopia: "markiewb" <benno.markiewicz@googlemail.com>
Ämne: Re: [nb-git-open-in-external-repoviewer] Additional patterns forgithub/gitblit (#24)

As the author of patts-qt, I'm curious how my repository location ended up here.
I'm not upset, but this page is the third DuckDuckGo result when searching "patts-qt", and I'm just wondering why it's mentioned here. This issue is pretty vague.
